README.md

```diff
@@ -75,6 +75,23 @@ $ gatsby develop
 ```
 6. Open webpage in http://localhost:8000.
 
+## How to Launch the Scraper
+
+```
+% cd data_pipeline/scraper 
+% virtualenv env
+% source env/bin/activate
```

Comment:
In Windows this is different, could you add:
./env/Scripts/activate as the windows version?

Reply:
Can I add a Windows section that says something like "TBD"? I have some access to Windows machines but not time to set up python development on them. It would be better if someone who uses Windows adds this, as they would do a better job.

Reply:
Works for me.